Samples/PublicSamples/Sample_README_Template.md

```diff
@@ -0,0 +1,27 @@
+Public Samples are created by developers who would like to share their features.
+
+Public Samples are not Official microsoft communication samples. Microsoft Communication does not support these samples. 
+Use them directly in Production is not recommended. Please consider contract Sample owner before consuming it in Production. 
```

Comment:
recommend:
Public Samples are provided by developers from the Microsoft Graph community.

Public Samples are not official Microsoft Communication samples, and not supported by the Microsoft Communication engineering team. It is recommended that you contact the sample owner before using code from Public Samples in production systems.

Reply:
Addressed comment in new iteration.